src/StarRatings/singlarobin/VanillaJS/index.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * Creates star rating functionality
+ * @param el DOM Element
+ * @param count Number of stars
+ * @param callback Returns selected star count to callback
+ */
+let lastClickedId = 0;
+
+function Star(el, count, callback) {
+    // write logic to create star rating utility.
+    const starWrapper = document.querySelector(el);
+    for (let i = 1; i <= count; i++) {
+        const element = document.createElement('i');
+        element.setAttribute('id', i);
+        element.classList.add('fa', 'fa-star-o');
+        starWrapper.appendChild(element);
+        element.addEventListener('click', (e) =>
```

Comment:
This is not a good approach to attach event listeners to each individual element, because if the count is big, it would take up a lot of memory to store the event listeners. Try using `event delegation` to solve this problem.